modules/utxo-lib/package.json

```diff
@@ -37,41 +43,21 @@
   ],
   "dependencies": {
     "@bitgo/blake2b": "^3.0.1",
-    "bech32": "0.0.3",
-    "bigi": "^1.4.0",
-    "bip66": "^1.1.0",
     "bitcoin-ops": "^1.3.0",
+    "bitcoinjs-lib": "git+https://github.com/BitGo/bitgo-utxo-lib.git#bitgo-bitcoinjs5",
```

Comment:
this commit is the only difference to upstream `master`: https://github.com/BitGo/bitgo-utxo-lib/commit/3ba50d18df7add35c5474ca392d033b42d83674e

Reply:
Is the plan to continue using `BitGo/bitgo-utxo-lib` or to revive the `BitGo/bitcoinjs-lib` repo? My only thought is that the latter makes it more clear that it's a fork of `bitcoinjs-lib` with minimal changes.

Reply:
ah yes https://github.com/BitGo/bitcoinjs-lib of course

thanks